thrift-interface-descriptions/service-cpis/profile-service/profile-user/profile-user-cpi.thrift

```diff
@@ -47,7 +47,11 @@ service UserProfileService {
   string initializeUserProfile (1: required security_model.AuthzToken authzToken)
                             throws (1: profile_user_cpi_errors.UserProfileServiceException upe,
                                     2: airavata_errors.AuthorizationException ae);
-
+  /**
+   * This method is deprecated and will be removed in future versions of our Django portal. Please use
```

Comment:
I would say "in future versions of Airavata".

Reply:
Sure, @machristie .
I will update it.

Reply:
I have made the requested change.